packages/styles/text-field-wrapper.css

```diff
@@ -2,9 +2,7 @@
   --text-field-wrapper-border-color: var(--field-border-color);
   --text-field-wrapper-background-color: var(--field-background-color);
   --text-field-wrapper-border-focus-color: var(--field-border-color-focus);
-  --text-field-wrapper-border-hover-color: var(
-    --field-border-color-focus-hover
-  );
```

Comment:
Just curious, why just this change for light mode and not dark mode?

Reply:
Actually, i didn't know if it applies on dark mode as well, since screenshot in the description related only to light mode, should it be applied on dark as well?

Reply:
Cauldron has to handle both light/dark modes, I'm assuming this was just a small oversight in the designs.

@shawnsharpDQ ideally we have somewhat consistent behaviors for both light and dark modes. Since we're making changes to inputs here do you have any recommendations on what we want to do to align here? It looks like dark mode has the focus ring color, is that the behavior we want to stick with?

Reply:
I would recommend to use `accent-light` for the hover color in dark theme. This is the same hover color we are using for secondary buttons, checkboxes, and radios.